x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/endpoint_details.tsx

```diff
@@ -126,7 +124,11 @@ export const EndpointDetails = memo(() => {
   return (
     <>
       {(show === 'policy_response' || show === 'isolate' || show === 'unisolate') && (
-        <EndpointDetailsFlyoutHeader hostname={hostDetails?.host?.hostname} />
+        <EndpointDetailsFlyoutHeader
+          hasBorder={true}
```

Comment:
nit: just `hasBorder` without the `={true}`

Reply:
53ad761a43d1aa0101049777c49270b398e8b08e